tests/testthat/test-Data-validity.R

```diff
@@ -6,5 +6,90 @@ test_that("validate_subjects returns TRUE for valid object", {
 })
 
 test_that("validate_subjects returns error message for non-valid object", {
-  expect_error(.GeneralData(ID = 1:3, cohort = 3:5, nObs = 4))
+  expect_error(.GeneralData(ID = 1:3, cohort = 3:5, nObs = 4), regexp = ".+")
```

Comment:
let's do like in helios - construct input object, make one single thing wrong/invalid, then run validate function on it, and expect specific error message